src/aui-tree/js/aui-tree-node.js

```diff
@@ -716,6 +716,13 @@ var TreeNode = A.Component.create({
         expand: function() {
             var instance = this;
 
+            var boundingBox = instance.get('boundingBox');
+            var nodeContainer = instance.get('container');
+
+            if (nodeContainer) {
+                boundingBox.append(nodeContainer);
+            }
```

Comment:
Could we add a check if the node is already appended? Something like 
```js
if (nodeContainer && !boundingBox.contains(nodeContainer)) {
```
That way we would revert to standard component behavior of show/hide after the first append.